CyberHealth/users/forms.py

```diff
@@ -94,3 +95,20 @@ def __init__(self, *args, **kwargs):
         for field in self.fields.values():
             field.error_messages = {'required': 'Enter {fieldname}'.format(
                 fieldname=field.label).capitalize()}
+
+
+class ForgotPasswordForm(PasswordResetForm):
+    error_messages = {}
+
+    email = forms.CharField(
+        required=True,
+        label="Email address",
+        help_text="Enter the email address you used for create an account for Assess your Cyber Health")
```

Comment:
Kindly amend to - "you used to create an account for"

Reply:
Addressed in the latest commit :)